backend/application/syson-application/src/test/java/org/eclipse/syson/application/controllers/diagrams/testers/DirectEditTester.java

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.syson.application.controllers.diagrams.testers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.jayway.jsonpath.JsonPath;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Consumer;
+
+import org.eclipse.sirius.components.collaborative.diagrams.dto.DiagramRefreshedEventPayload;
+import org.eclipse.sirius.components.collaborative.diagrams.dto.EditLabelInput;
+import org.eclipse.sirius.components.collaborative.diagrams.dto.EditLabelSuccessPayload;
+import org.eclipse.sirius.components.diagrams.Diagram;
+import org.eclipse.sirius.components.diagrams.InsideLabel;
+import org.eclipse.sirius.components.diagrams.tests.graphql.EditLabelMutationRunner;
+import org.eclipse.sirius.components.diagrams.tests.navigation.DiagramNavigator;
+
+import reactor.test.StepVerifier.Step;
+
+/**
+ * Tester that checks the initial label used as input of the DirectEdit tool.
```

Comment:
seems to be the wrong description (it is the same than DirectEditInitialLabelTester)

Reply:
Done.